hugo/layouts/papers/single.html

```diff
@@ -336,6 +588,9 @@ <h5 class="card-title">Abstract</h5>
         <span class="px-1">Video</span>
       </a>
       {{ end }}
+      <a class="btn btn-warning d-flex flex-wrap justify-content-center" href=# title="Correct problems with title, author list, and abstract" onclick="showMetadataDialog()">
+        <span class="pl-sm-2 d-none d-sm-inline">Fix metadata</span>
```

Comment:
Remove all the CSS classes here, they only make sense if there is an icon in front of the text

(alternatively, add a pen icon? :))

Reply:
A pen icon it is!